lib/declare_schema/model.rb

```diff
@@ -128,13 +128,17 @@ def belongs_to(name, scope = nil, **options, &block)
 
         fk = options[:foreign_key]&.to_s || "#{name}_id"
 
-        if !options.has_key?(:optional) && Rails::VERSION::MAJOR >= 5
+        if !options.has_key?(:optional)
           options[:optional] = column_options[:null] # infer :optional from :null
         end
 
         fk_options[:dependent] = options.delete(:far_end_dependent) if options.has_key?(:far_end_dependent)
 
-        super(name, scope, options)
+        if Rails::VERSION::MAJOR >= 5
+          super(name, scope, options)
```

Comment:
I don't see anything using the provided block. Can we get rid of it and just call `super` here?

Reply:
Good point. I've removed `&block` and changed this to plain `super`.